src/commands/install.rs

```diff
@@ -98,6 +105,38 @@ impl super::command::Command for Install {
         let version_str = format!("Node {}", &version);
         outln!(config#Info, "Installing {} ({})", version_str.cyan(), safe_arch.to_string());
 
+        /*
+        Prepare reinstalling packages.
+         */
+        let install_packages_string_option = (match self.reinstall_packages_from {
+            Some(user_version) => {
+                let output = std::process::Command::new("sh")
+                    .arg("-c")
+                    .arg(format!(
+                        concat!(
+                        "fnm use \"{}\" >/dev/null ",
+                        "&& npm list -g --depth=0 2>/dev/null ",
+                        "| command sed 1,1d ",
+                        "| command sed ",
+                        "-e '/ -> / d' ",
+                        "-e '/\\(empty\\)/ d' ",
+                        "-e 's/^.* \\(.*@[^ ]*\\).*/\\1/' ",
+                        "-e '/^npm@[^ ]*.*$/ d'",
+                        ),
+                        user_version,
+                    ))
+                    .output().expect("Failed to get package list.");
```

Comment:
This will not work on Windows-based shells (wincmd).
Maybe we can try adding a new command, `fnm reinstall-packages-from`, and we can add unit/e2e tests for it.